packages/server/graphql/mutations/helpers/notifications/MSTeamsNotifier.ts

```diff
@@ -43,6 +44,45 @@ const notifyMSTeams = async (
 }
 export type MSTeamsNotificationAuth = IntegrationProviderMSTeams & {userId: string}
 
+const createTeamPromptMeetingTitle = (meeting: Meeting) => `*${meeting.name}* is open 💬`
+
+const GENERIC_MEETING_TITLE = `Meeting Started 👋`
+
+const MeetingTypeTitleLookup: Record<MeetingTypeEnum, ((meeting: Meeting) => string) | string> = {
+  action: GENERIC_MEETING_TITLE,
+  poker: GENERIC_MEETING_TITLE,
+  retrospective: GENERIC_MEETING_TITLE,
+  teamPrompt: createTeamPromptMeetingTitle
+}
+
+const createGenericMeetingAction = (meetingUrl: string) => {
+  const joinMeetingAction = new AdaptiveCards.OpenUrlAction()
```

Comment:
:scream: did...did a C# dev write this whole package? goodness gracious look at those class extensions & nullary constructors! no action needed, just venting :slightly_smiling_face:

Reply:
yeah, I was thinking the same 😅